packages/ui-extensions/src/surfaces/customer-account/components/ImageGroup/ImageGroup.ts

```diff
@@ -0,0 +1,48 @@
+import {createRemoteComponent} from '@remote-ui/core';
+import type {RemoteFragment} from '@remote-ui/core';
+
+export interface ImageGroupProps {
+  /**
+   * The layout of the images.
+   * When set to "grid", the images will be displayed in a grid layout with max images of 4.
+   * When set to "inline-stack", the images will be displayed in a single row with max images of 3.
+   *
+   * @default "grid"
+   */
+  variant?: 'grid' | 'inline-stack';
+
+  /**
+   * Accessibility label for the image group.
+   */
+  imagesAccessibilityLabel?: string;
+
+  /**
+   * Loading state of the component.
+   * @default false
+   */
+  loading?: boolean;
+
+  /**
+   * Images to be displayed.
+   *
+   */
+  images: RemoteFragment;
+
+  /**
+   * Total count of items.
```

Comment:
Worth to mention that if `totalItems` is less than the amount of images passed we are going to use that instead?

Reply:
Think so as well and have updated the description